test/fixtures/test-jest-app/stats.js

```diff
@@ -0,0 +1,19 @@
+const fs = require('fs');
+
+const statsFile = __dirname + '/stats.txt';
+
+function getCount() {
+  return parseInt(fs.readFileSync(statsFile), 10);
+}
+
+module.exports.incrementCount = function() {
+  let count = getCount();
+  count += 1;
+  fs.writeFileSync(statsFile, String(count));
+}
+
+module.exports.getCount = getCount;
+
+module.exports.resetCount = function() {
+  fs.writeFileSync(statsFile, '0');
```

Comment:
I think one reason for parsing stdout instead of writing to a file is if in the future, JSDom is replaced with headless chrome, we can't really write to the filesystem and we'd need to change how these tests work.